config/README.md

```diff
@@ -0,0 +1,20 @@
+# Features File
+
+ The feature.json file is used for easily managing code that we want to opt in or opt out of, such as code involving the Content Management System (CMS).
+
+ ## Purpose
+
+ The purpose of the feature.json file is to provide a centralized location where developers can enable or disable specific code features based on their requirements. This allows for greater flexibility and control over the functionality of the software.
+
+ ## How to Use
+
+ To use the features file, follow these steps:
+
+ 1. Open the features file located at `config/feature.json`.
+ 2. Locate the section related to the specific feature you want to enable or disable.
+ 3. Set the corresponding flag to `true` or `false` to opt in or opt out of the feature.
+ 4. Save the file.
+
+ By using the feature.json file, developers can easily manage and control the code features without the need for extensive code modifications or deployments.
```

Comment:
Change to: developers can integrate new functionality in a controlled way, earlier in the development cycle. This functionality can be enabled once appropriate testing and acceptance has been performed.

Remove the bit about extensive code modifications or deployments because we will still need to modify code and this feature optioning does require a build and deployment of the website to take effect.  This is where ai generated docs need a careful bit of reviewing.